src/EditorFeatures/Core.Wpf/Suggestions/SuggestedActionsSource.cs

```diff
@@ -897,6 +897,11 @@ private void OnSuggestedActionsChanged(Workspace currentWorkspace, DocumentId cu
                 using (var asyncToken = _owner.OperationListener.BeginAsyncOperation(nameof(GetSuggestedActionCategoriesAsync)))
                 {
                     var document = range.Snapshot.GetOpenDocumentInCurrentContextWithChanges();
+                    if (document == null)
```

Comment:
This change was merged as a part of #24937 but I had it in my PR too to validate locally.